### ISSUE TITLE
`Internationaler Frauentag` missing for German MV

### ISSUE THREAD

**Issue by mblumtritt** [holidays/definitions#240]
The `Internationaler Frauentag` is a bank holiday in Germany not only in Berlin but since 2023 also in Mecklenburg-Vorpommern.

This definition is missing.

Found in [German wikipedia](https://de.wikipedia.org/wiki/Internationaler_Frauentag#Der_Frauentag_als_gesetzlicher_Feiertag)

**Comment by jiveeee** [comment]
Yes. would you please update this missing holiday for Mecklenburg-Vorpommern? I appreciate it. :)

**Comment by dennisvandehoef** [comment]
This one can be closed, Was done in https://github.com/holidays/definitions/pull/245

**Comment by ppeble** [comment]
Fixed in #245 back in January 2024, this just never got closed. Thanks for the nudge @dennisvandehoef.

Verified against current master:

    2022 de_mv: []
    2023 de_mv: ["Internationaler Frauentag"]
    2024 de_mv: ["Internationaler Frauentag"]

Berlin from 2019 and Mecklenburg-Vorpommern from 2023, both correct. Added a boundary test for the 2022 case in #372 while I was here.
